animation_nodes/nodes/sound/MIDI_Note_Evaluate.py

```diff
@@ -0,0 +1,61 @@
+import bpy
+from ... data_structures import DoubleList
+from ... base_types import AnimationNode, ListTypeSelectorSocket, VectorizedSocket
+
+def filterByNote(self, notesIn, noteNumber, time, attackTime, attackInterpolation, releaseTime, releaseInterpolation):
```

Comment:
This should be a class method. And why is it called `filterByNote`?

Reply:
Created this function when the channel and note filter are optional. But I can reintegrate the code in the class part if you think is better.

Reply:
Yes, please do.